### PR TITLE
Added curl call to auto-release.yml to be created in simple-icons-pdf

### PR DESCRIPTION
**Pull request by jorgeamadosoria**
Partial solution for the automatic release of the PDF repo.
This has been mentioned in several issues in several repos.
**Issues:**: 
#5804 

This depends on https://github.com/simple-icons/simple-icons-pdf/pull/2

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -101,6 +101,17 @@ jobs:
...
+  pdf:
+    name: Trigger simple-icons-pdf release
+    needs: npm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger simple-icons-pdf release
+        run: |
+          curl -X POST \
+            -H "Authorization: Bearer ${{ secrets.REMOTE_DISPATCH_TOKEN }}" \
+            -d '{"ref":"develop"}' \
```
Comment (ericcornelissen) [inline .github/workflows/publish.yml:113]: At this stage this should be `master`, [simple-icons-repo](https://github.com/simple-icons/simple-icons-pdf/branches) does not have a `develop` branch.

```suggestion
            -d '{"ref":"master"}' \
```

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -101,6 +101,17 @@ jobs:
             -H "Authorization: Bearer ${{ secrets.REMOTE_DISPATCH_TOKEN }}" \
             -d '{"ref":"develop"}' \
             https://api.github.com/repos/simple-icons/simple-icons-font/actions/workflows/auto-release.yml/dispatches
+  pdf:
+    name: Trigger simple-icons-pdf release
+    needs: npm
```
Comment (ericcornelissen) [inline .github/workflows/publish.yml:106]: It probably makes more sense to make this depend on `sanity-check`, this is not technically dependent on the `npm` release.

```suggestion
    needs: sanity-check
```

(same for the `font` job)

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -101,6 +101,17 @@ jobs:
...
+    steps:
+      - name: Trigger simple-icons-pdf release
+        run: |
+          curl -X POST \
+            -H "Authorization: Bearer ${{ secrets.REMOTE_DISPATCH_TOKEN }}" \
+            -d '{"ref":"develop"}' \
+            https://api.github.com/repos/simple-icons/simple-icons-pdf/actions/workflows/auto-release.yml/dispatches
   website:
     name: Trigger simple-icons-website update
     needs: npm
```
Comment (ericcornelissen) [inline .github/workflows/publish.yml:117]: This should probably depend on the new `pdf` job as well

```suggestion
    needs: [npm, pdf]
```

**Review by ericcornelissen — CHANGES_REQUESTED** [review]
Some preliminary requests for changes

**Comment by adamrusted** [comment]
Closing, as the PDF repo is being archived.

**Pull request closed without merging**